README.md

```diff
@@ -458,6 +458,41 @@ We use it for the control container because it needs to be available early to gi
 
 Be careful, and make sure you have a similar low-level use case before reaching for host containers.
 
+#### Bootstrap containers settings
+* `settings.bootstrap-containers.<name>.source`: The URI of the container
+* `settings.bootstrap-containers.<name>.mode`: The mode of the container, it could be one of `off`, `once` or `always`
+* `settings.bootstrap-containers.<name>.user-data`: field with arbitrary base64-encoded data
+
+Bootstrap containers are host containers that can be used to "bootstrap" the host before services like ecs-agent, kubelet and docker start.
+
+This type of containers are very similar to normal host containers, they come with persistent storage and with optional user data.
+Unlike normal host containers, bootstrap containers can't be treated as `superpowered` containers.
+However, these containers run as privileged and have access to the underlying root filesystem on `/.bottlerocket/rootfs`.
+Bootstrap containers are started once the `configured` systemd's target is reached, so they are not executed in a deterministic order.
```

Comment:
I updated the documentation, and I mentioned that bootstrap containers run in parallel. Regarding the deterministic order, I think use cases that require specific order should belong to one container instead of splitting work and orchestrating containers. Do you have an specific use case in mind?